packages/pieces/openai/src/lib/actions/translation.ts

```diff
@@ -0,0 +1,48 @@
+import { HttpRequest, HttpMethod, httpClient } from "@activepieces/pieces-common";
+import { Property, createAction } from "@activepieces/pieces-framework";
+import { openaiAuth } from "../..";
+import FormData from "form-data";
+import mime from 'mime-types';
+
+export const translateAction = createAction({
+    name: 'translate',
+    displayName: 'translate',
+    description: 'translate audio to text using whisper-1 model',
+    auth: openaiAuth,
+    props: {
+        audio : Property.File({
+            displayName: 'Audio',
+            required: true,
+            description: 'Audio file to translate',
+        }),
+        fileName : Property.ShortText({
```

Comment:
You don't really need this, it won't effect the result, you can just use the name in audio property.